pkg/manager/impl/validation/execution_validator.go

```diff
@@ -16,7 +16,8 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
-const allowedExecutionNameLength = 20
+// Maximum value length of a Kubernetes label
+const allowedExecutionNameLength = 63
```

Comment:
should we also update the name length for [generated names](https://github.com/flyteorg/flyteadmin/blob/master/pkg/common/executions.go#L11)?

Reply:
Defer to other maintainers, seems OK to leave the randomly generated string short (since a longer name might make the frontend less legible). If it's preferable to have both be consistent, I'm happy to make the change (cc @kumare3 @eapolinario)

Reply:
sure, just wanted to check if the omission was intentional :)
I think the chance of collisions for the 20 char generated name isn't a concern, so keeping as is sounds good 👍